CMakeLists.txt

```diff
@@ -205,21 +206,25 @@ EndIf(FAIRSOFT_EXTERN)
 if(NOT DEFINED Boost_NO_SYSTEM_PATHS)
   Set(Boost_NO_SYSTEM_PATHS TRUE)
 endif()
-Set(Boost_NO_BOOST_CMAKE TRUE)
+#Set(Boost_NO_BOOST_CMAKE TRUE)
```

Comment:
Here I would prefer something like

```cmake
if(NOT DEFINED Boost_NO_BOOST_CMAKE)
  set(Boost_NO_BOOST_CMAKE TRUE)
endif()
```

That way you can override our default in the alidist recipe via `-D`.

Reply:
Ok.